### ISSUE TITLE
Notification icons not working

### ISSUE THREAD

**Issue by MattHoneycutt** [creativetimofficial/material-dashboard-pro-angular#109]
The bootstrap-notify plugin does not show icons correctly.  You can view the demo/documentation here:

http://md-pro-angular.creative-tim.com/documentation/notifications

The example JS code shows this:

```
$.notify({
icon: "add_alert",
message: "Welcome to <b>Material Dashboard Pro</b> - a beautiful dashboard for every web developer."

},{
type: 'success',
timer: 4000,
placement: {
from: from,
align: align
}
});
}
```

But the icon is not actually visible.  Any suggestions?

**Comment by MattHoneycutt** [comment]
Any update on this?   Anyone else come up with a workaround to show notifications?

**Comment by chelaruc** [comment]
@MattHoneycutt Thank you for using our product! We'll do an update next week and we will fix that.

**Comment by PavelValentov** [comment]
So it does not still working.
Have you any news?

**Comment by Se0ng11** [comment]
is still not working, r the plugin being abandoned?

**Comment by chelaruc** [comment]
Fixed in v2.0.0
